llvm/include/llvm/CodeGen/AccelTable.h

```diff
@@ -360,6 +360,49 @@ class DWARF5AccelTable : public AccelTable<DWARF5AccelTableData> {
     unsigned Index;
     DWARF5AccelTableData::AttributeEncoding Encoding;
   };
+  union AbbrevDescriptor {
+    struct {
+      uint32_t CompUnit : 1;
+      uint32_t TypeUnit : 1;
+      uint32_t DieOffset : 1;
+      uint32_t Parent : 2;
+      uint32_t TypeHash : 1;
+      uint32_t Tag : 26;
+    } Bits;
+    uint32_t Value = 0;
```

Comment:
Could we avoid the union entirely, and use the structure with the bitfield members directly instead?

Reply:
Changed it struct. Although it requires bit_cast and memset. Unless there is some c++ I am not aware of.